travispy/entities/_entity.py

```diff
@@ -192,8 +192,7 @@ def _load(cls, infos, session):
                 try:
                     setattr(entity, key, value)
                 except AttributeError:
-                    warn('Unknown {0} attribute {1}'
-                         .format(entity.__class__.__name__, key))
+                    pass
```

Comment:
Instead of `log.warning` you can change it to `log.debug` so these messages will appear only it log is set to DEBUG level (which is not the default). The final result you be no messages unless user explicitly change the log level. What do you think of that?